packages/aml-checklist/src/reducers/auth.ts

```diff
@@ -3,6 +3,7 @@ import { isType } from '@/utils/actions'
 import { authLogin, authLoginFailure, authLoginSuccess, authLogoutSuccess, authSetRefreshSession } from '@/actions/auth'
 import { RefreshParams, LoginSession, getSessionCookie } from '@reapit/cognito-auth'
 import { COOKIE_SESSION_KEY_AML_APP } from '../constants/api'
+import config from '../../config.json'
```

Comment:
I think cannot import config.json directly because when async load happen will cause app crash

Reply:
Got your idea, I'm fixing it